crawl-ref/source/tile-flags.h

```diff
@@ -48,6 +48,8 @@ enum tile_flags ENUM_INT64
     TILE_FLAG_PINNED       = 0x200000000000000ULL,
     TILE_FLAG_VILE_CLUTCH  = 0x400000000000000ULL,
     TILE_FLAG_POSSESSABLE  = 0x800000000000000ULL,
+    TILE_FLAG_MORE_POISON  = 0x1000000000000000ULL,
+    TILE_FLAG_MAX_POISON   = 0x2000000000000000ULL,
```

Comment:
You only need to add a single flag, since there are four possible states (no poison, and poison 1-3), and four states can be represented with two bits. You'd need to change the rest of the code that sets and checks these flags.

An alternative that I'd accept is just to leave a comment above these saying that if we run out of space, these two can be combined.

Reply:
I did the alternative you suggested for now.